spec/errors/invalidTypeSpec.js

```diff
@@ -0,0 +1,10 @@
+var invalidType = require( "../../js/errors/invalidType.js" );
```

Comment:
It is best to keep the variable name consistent with the variable name in the `invalidType` file, so this should probably be `InvalidTypeError`.

Also, we support ES2015 in the unit tests so you can use the new import syntax:
```js
import InvalidTypeError from "../../js/errors/invalidType";
```